torch/csrc/jit/python/python_arg_flatten.cpp

```diff
@@ -39,12 +39,6 @@ py::object cast_handle_sequence(std::vector<py::handle> objs) {
 }
 
 void flatten_rec(PyObject* obj, ParsedArgs& args) {
-  auto as_variable = [](at::Tensor& tensor) // Wrap tensor as Variable
-  {
-    PyObject* wappred_obj = THPVariable_Wrap(tensor);
```

Comment:
Is it ok to remove the `THPVariable_Wrap` call here? Does it has any side effect that was important?

Reply:
So, this old code wrapped the tensor in a PyObject Variable, and then *immediately* unwrapped it back into a Tensor. The only side effect of this, so far as I can tell, is that it ensures that the pyobj is set. But this is pointless as we always ensure there is a valid pyobj when doing `THPVariable_wrap`. 

The code was added in #54864 without any serious code review (cc @SplitInfinity) so my guess is it was just cargo culted from somewhere else.

Reply:
Sounds good!